### ISSUE TITLE
real application for annotating cell types

### ISSUE THREAD

**Issue by yueming-ding** [bowang-lab/scGPT#55]
Hi I would like to discuss with you on how to annotate cell types for new data. Typically, our new single cell data doesn't have cell annotations and only have gene expression data. So if we want to apply scGPT to annotate the new data, it is not possible to finetune scGPT with our new data since we don't have any celll annotations for this data. What is the best workflow to annotate the new dataset with scGPT if the new dataset doesn't have any cell type annotation info? Thanks!

**Comment by yueming-ding** [comment]
@subercui Thank so much you for your answering and providing a new notebook! We are testing your new notebook. But could you provide the link to the following reference dataset and test dataset so that we can test it?
adata = sc.read_h5ad("../data/annotation_pancreas/demo_train.h5ad")
test_adata = sc.read_h5ad("../data/annotation_pancreas/demo_test.h5ad")

I think reference mapping mainly relies on what kind of reference you use. It is a challenging to assign every new dataset with an appropriate reference. If you could use all the cells on CellXGene as a reference atlas and update the reference with your new data, that will be great! I look forward to your new reference with all the cells on CellXGene and corresponding new notebook. We'll love to do further test with this new reference.

Many thanks again!!!

**Comment by subercui** [comment]
Hi @yueming-ding , thanks for pointing it out! We have uploaded the notebook with the link to data. We have also been working on the reference atlas mapping notebook, which is close to complete. We will aim to upload it tonight!

**Comment by yueming-ding** [comment]
> Hi @yueming-ding , thanks for pointing it out! We have uploaded the notebook with the link to data. We have also been working on the reference atlas mapping notebook, which is close to complete. We will aim to upload it tonight!

@subercui Yes, I found the link to those data. Thanks for your updating! I didn't find your new reference atlas mapping notebook. Please let me know when it is available. Thanks!!!

**Comment by subercui** [comment]
Hi @yueming-ding , thank you for your patience. We have released the reference mapping tutorial at https://github.com/bowang-lab/scGPT/blob/main/tutorials/Tutorial_Reference_Mapping.ipynb . Please let me know your feedback!

**Comment by yueming-ding** [comment]
> Hi @yueming-ding , thank you for your patience. We have released the reference mapping tutorial at https://github.com/bowang-lab/scGPT/blob/main/tutorials/Tutorial_Reference_Mapping.ipynb . Please let me know your feedback!

@subercui Thank you so much! We'll test it and let you our test results.
